### PR TITLE
Feature: add function getSwapEnabled

### PR DESCRIPTION
**Pull request by Zitzak**
- Implements #217 

closes #127

### CODE REVIEW

File: contracts/lbp/LBPManager.sol [hunk 1]
```diff
@@ -305,6 +305,13 @@ contract LBPManager {
         lbp.setSwapEnabled(_swapEnabled);
     }
 
+    /**
+     * @dev                              Tells whether swaps are enabled or not for the given pool
+     */
+    function getSwapEnabled() public view returns (bool) {
```
Comment (codebuster22) [inline contracts/lbp/LBPManager.sol:311]: we can make it external, but it's fine.

**Review by codebuster22 — DISMISSED** [review]
lgtm

File: contracts/lbp/LBPManager.sol [hunk 2]
```diff
@@ -305,6 +305,13 @@ contract LBPManager {
         lbp.setSwapEnabled(_swapEnabled);
     }
 
+    /**
+     * @dev                              Tells whether swaps are enabled or not for the given pool
```
Comment (codebuster22) [inline contracts/lbp/LBPManager.sol:309]:
```suggestion
     * @dev                              Tells whether swaps are enabled or not for the LBP
```

**Review by codebuster22 — DISMISSED** [review]
lgtm

File: contracts/lbp/LBPManager.sol [hunk 3]
```diff
@@ -305,6 +305,13 @@ contract LBPManager {
         lbp.setSwapEnabled(_swapEnabled);
     }
 
+    /**
+     * @dev                              Tells whether swaps are enabled or not for the LBP
+     */
+    function getSwapEnabled() public view returns (bool) {
```
Comment (codebuster22) [inline contracts/lbp/LBPManager.sol:311]:
```suggestion
    function getSwapEnabled() external view returns (bool) {
```

**Review by fabianschu — APPROVED** [review]

**Review by codebuster22 — APPROVED** [review]
lgtm

**Pull request merged**